test/relay.test.js

```diff
@@ -0,0 +1,323 @@
+'use strict';
+
+var chai = require('chai')
+  , expect = chai.expect
+  , resolver = require('../src/resolver')
+  , helper = require('./helper')
+  , sequelize = helper.sequelize
+  , Sequelize = require('sequelize')
+  , Promise = helper.Promise
+  , attributeFields = require('../src/attributeFields');
+
+import {
+  GraphQLString,
+  GraphQLInt,
+  GraphQLFloat,
+  GraphQLNonNull,
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLList,
+  GraphQLObjectType,
+  GraphQLSchema,
+  graphql
+} from 'graphql';
+
+import {
+  sequelizeNodeInterface
+} from '../src/relay';
+
+import {
+  globalIdField,
+  connectionDefinitions,
+  connectionArgs,
+  connectionFromArray
+} from 'graphql-relay';
+
+function generateTask(id) {
+  return {
+    id: id,
+    name: Math.random().toString()
+  }
+}
+
+describe('relay', function () {
+  var User
+    , Task
+    , userType
+    , taskType
+    , taskConnection
+    , nodeInterface
+    , Project
+    , projectType
+    , userConnection
+    , nodeField
+    , schema;
+
+  before(function () {
+    sequelize.modelManager.models = [];
+    sequelize.models = {};
+    User = sequelize.define('user', {
+      name: {
+        type: Sequelize.STRING
+      }
+    }, {
+      timestamps: false
+    });
+
+    Task = sequelize.define('task', {
+      name: {
+        type: Sequelize.STRING
+      }
+    }, {
+      timestamps: false
+    });
+
+    Project = sequelize.define('name', {
+      name: {
+        type: Sequelize.STRING
+      }
+    }, {
+      timestamps: false
+    });
+
+    User.Tasks = User.hasMany(Task, {as: 'tasks'});
+    Project.Users = Project.hasMany(User, {as: 'users'});
+
+    var node = sequelizeNodeInterface(sequelize, {Project: projectType, User: userType, Task: taskType});
```

Comment:
The object is necessary, unfortunately. sequelizeNodeInterface tells graphql how to determine which graphql type to association with an object, and the only way to do that is by mapping Model names to the types that are defined for them.